person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(age, parent_permission: true, name: 'unknown')
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_accessor :name, :age
+  attr_reader :id
```

Comment:
- [ ] **Optional:**

Great job coming this far!

The `attr` is conventionally defined at the class before the `initialize()` method. Note this in future projects.